File: Lecture 002 Recursion Phod Denge/Head.java
import java.util.*;
 
public class Head {
 

static void func(int n)
{
 //Base Case
 if(n==0) 
    return ;

 //Bharosa(trust)
 func(n-1);
 
 //Tumhara kaam
 System.out.println(n);
}
 
// Driver code
public static void main(String[] args)
{
    int n;
    Scanner sc=new Scanner(System.in);  
    n= sc.nextInt();
 
    func(n);
}
}

File: Lecture 002 Recursion Phod Denge/Head.cpp
#include<iostream> 
using namespace std;

void func(int n){
    // Base case
    if(n==0) 
        return ;
    
    //Bharosa(trust)
    func(n-1);

    //tumhara kam
    cout<<n<<endl;
}
int main(){
    int n;
    cin>>n;
    func(n);
}